source/androidx.media3/media3-common/Transforms/Metadata.xml

```diff
@@ -18,17 +18,24 @@
         Java.Lang.Object
     </attr>
 
-
     <attr
         path="/api/package[@name='androidx.media3.common']/interface[@name='Player.Listener']/method[@name='onCues' and count(parameter)=1 and parameter[1][@type='java.util.List&lt;androidx.media3.common.text.Cue&gt;']]"
         name="managedName"
         >
         OnCuesDeprecated
-    </attr>
-    <attr
-        path="/api/package[@name='androidx.media3.common']/interface[@name='Player.Listener']/method[@name='onPositionDiscontinuity' and count(parameter)=1 and parameter[1][@type='int']]"
-        name="managedName"
-        >
-        OnPositionDiscontinuityDeprecated
-    </attr>
+	  </attr>
+
+    <!-- Remove these as they cause issues with Media3.ExoPlayer-->
+    <remove-node
```

Comment:
So there's these 2 methods in the binding:

```java
@Deprecated
default void onPositionDiscontinuity(int reason) { ... }

default void onPositionDiscontinuity(PositionInfo oldPosition, PositionInfo newPosition, int reason) { ... }
```

[Source](https://developer.android.com/reference/androidx/media3/common/Player.Listener#onPositionDiscontinuity(int))

If we're going to remove one of them, it seems like we would want to remove the deprecated one and leave its replacement.

Reply:
I will update that and test